Rock.Rest/Controllers/ContentChannelItemsController.Partial.cs

```diff
@@ -0,0 +1,76 @@
+﻿// <copyright>
+// Copyright by the Spark Development Network
+//
+// Licensed under the Rock Community License (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.rockrms.com/license
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+//
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Http;
+using System.Web.Http.OData;
+
+using Rock.Data;
+using Rock.Model;
+using Rock.Rest.Filters;
+using Rock.Web.Cache;
+
+namespace Rock.Rest.Controllers
+{
+    /// <summary>
+    /// ContentChannelItems REST API
+    /// </summary>
+    public partial class ContentChannelItemsController
+    {
+        #region ContentByDataViewGuids
+        /// <summary>
+        /// Returns a list of content channel items based on a list of dataview guids
+        /// </summary>
+        /// <param name="guids"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [EnableQuery]
+        [Authenticate, Secured]
+        [System.Web.Http.Route( "api/ContentChannelItems/GetFromPersonDataView" )]
+        public IQueryable<ContentChannelItem> GetFromPersonDataView( string guids )
+        {
+            RockContext rockContext = new RockContext();
```

Comment:
So there's a RockContext type and a RockContext Class? hmmm. +1 to why I don't like strong typing

Reply:
😂